.github/workflows/pr-title.yml

```diff
@@ -2,8 +2,6 @@ name: Check Conventional Commits format
 
 on:
   pull_request_target:
-    branches:
```

Comment:
Is this necessary? Not convinced, but if so, we should clarify what the conventions are for PRs to other branches -- e.g. are they supposed to signal fixes or breaking changes to the branch?